src/Scenes/Game/GameLogView.tsx

```diff
@@ -10,23 +10,42 @@ export interface GameLogProps {
 
 export default function GameLogView({ logs }: GameLogProps) {
     const messagesEnd = useRef<HTMLDivElement>(null);
+    const logBoxRef = useRef<HTMLDivElement>(null);
+    const isAtBottomRef = useRef(true);
+    const [prevLogsLength, setPrevLogsLength] = useState(logs.length);
 
     const [isLogOpen, setIsLogOpen, gameLogRef] = useCloseOnLoseFocus<HTMLDivElement>();
 
+    const checkIfAtBottom = () => {
```

Comment:
Made the following changes:

1. Added `useCallback` to the import statement from `react`.
2. Wrapped the `checkIfAtBottom` function with `useCallback`
3. Added an empty dependency array `[]` to the `useCallback` hook since this function doesn't depend on any props or state

Reply:
this one is ok